configs/eslint-plugin-devtools/rules/no-expect-method-without-call.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const PROPERTY_TERMINATORS = [
+  'ok',
+  'true',
+  'false',
+  'null',
+  'undefined',
+  'exist',
+  'empty',
+  'arguments',
+];
+
+function followNodeChainToExpectCall(node) {
+  while (node) {
+    if (node.type === 'CallExpression') {
+      if (node.callee.type === 'Identifier' && node.callee.name === 'expect') {
+        return node;
+      }
+      if (node.callee.type === 'MemberExpression') {
+        node = node.callee.object;
+        continue;
+      }
+    }
+
+    // Continue on the node chain (e.g. .not, .to).
+    if (node.type === 'MemberExpression') {
+      node = node.object;
+      continue;
+    }
+
+    break;
+  }
+}
+
+/** @type {import('eslint').Rule.RuleModule} */
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description:
+        'Require invocation of expect method assertions (e.g., to.throw())',
+    },
+    messages: {
+      methodNotInvoked:
+        'expect().to.[METHOD] must be invoked with parentheses, e.g., expect(() => someFn()).to.throw()',
```

Comment:
Now that its generalized we should make the hint say the actual method, and we could see if a hint for adding the `()` is worthwhile. I'll update that tomorrow.